build.gradle

```diff
@@ -320,7 +305,7 @@ def listEmbedDependencies = { rootModuleName, prefix ->
         }
     }
     if (list.isEmpty()) {
-        throw new GradleException('Failed to collect libraries to embed: no dependencies found for: "' + rootModuleName + '"')
+        throw new GradleException('Failed to collect libraries to embed: no dependencies found')
```

Comment:
`rootModuleName` is not useful anymore?

Reply:
Ah, yeah, maybe. I kept it for now just because it is still used in the condition `firstLevelDependency.moduleName == rootModuleName`, and I wanted to make this change as small as possible (it's already big enough).

We'll want to revisit the enumeration logic, and simplify it more.